README.md

```diff
@@ -26,8 +26,9 @@ Download the latest romero.gateway version which is compatible with your OMERO s
 
 OMERO version | romero.gateway version
 --- | --- 
+5.6 | 0.4.8
```

Comment:
Does it also work with 5.5?

Reply:
I'm not sure, that's why I just put 5.6 there. But I'll test it, could be less restrictive then.

Reply:
It is compatible to 5.5. Updated Readme accordingly again.